src/app/components/service-down/service-down.component.html

```diff
@@ -1,8 +1,10 @@
 <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
-      <p class="govuk-body">Try again later.</p>
+      <h1 class="govuk-heading-xl">{{(currentError)?.header}}</h1>
+      <p class="govuk-body" *ngFor="let error of currentError.errors">
+        <a *ngIf="error.urlText" href="{{error.url}}">{{error.urlText}}</a> {{ error.bodyText }}
```

Comment:
might want to use routerLink instead of href

Reply:
Done